table/run_queries.py

```diff
@@ -27,21 +28,48 @@
 
 client = cloud_bigquery.Client(project=firehook_resources.PROJECT_NAME)
 
+SOURCE_PLACEHOLDER = 'SOURCE'
+SINK_PLACEHOLDER = 'SINK'
 
-def run_query(filepath: str) -> cloud_bigquery.table.RowIterator:
+DEFAULT_SOURCE_DATASET = 'base'
+DEFAULT_SINK_DATASET = 'derived'
+
+
+def _run_query(filepath: str, source: str,
```

Comment:
Let's use `base_dataset` and `derived_dataset` to be clearer

Reply:
done